internal/ui/src/components/empty.tsx

```diff
@@ -0,0 +1,72 @@
+import type * as React from "react";
+
+import { cn } from "../lib/utils";
+
+interface EmptyRootProps extends React.HTMLAttributes<HTMLDivElement> {
+  fill: boolean | undefined;
```

Comment:
```suggestion
  fill?: boolean;
```

This way it's also optional but it doesn't complain if you omit it